sql/modules/Reconciliation.sql

```diff
@@ -520,6 +562,18 @@ $$
                 max_ac_id = (select max(entry_id) from acc_trans)
         where id = in_report_id;
 
+    -- How can we get already cleared transactions?
```

Comment:
Please don't put the questions but the answers in the comments :-) !

Reply:
We are testing (defensively?) for a condition that should never happen in the application. Cleared transactions can only arise from a reconciliation, yet the test data provide some.

Reply:
I'm not sure about your reaction. Is that the response to the question in the code? If so, then shouldn't the response above be in the code?

Reply:
also, if they can't occur in the application, maybe we should adjust the migration so that they can't happen in the migration either?

Reply:
The problem isn't migration. Test data seems designed to provide an invalid situation, which we took for ok for many years.

Reply:
Ok. But the original remark remains: don't put questions, but answers in the code comments.